src/action-ros-ci.ts

```diff
@@ -357,10 +357,16 @@ async function run_throw(): Promise<void> {
 
 	const options: im.ExecOptions = {
 		cwd: rosWorkspaceDir,
+		env: {
+			...process.env,
+			ROS_VERSION: targetRos1Distro ? "1" : "2",
+			ROS_PYTHON_VERSION:
+				targetRos1Distro && targetRos1Distro != "noetic" ? "2" : "3",
```

Comment:
This is rather nitpicky detail (:sweat_smile:), but we technically support providing both `target-ros1-distro` and `target-ros2-distro`. I think the only use-case for that is building the ros1_bridge: https://github.com/ros-tooling/action-ros-ci/blob/7ff80b89366718de2a315b5e9d9201edb81667a4/action.yml#L64-L65. For that use-case, I think the values for `ROS_VERSION`/`ROS_PYTHON_VERSION` should be `2`/`3` (respectively) because `target-ros1-distro` would only be defined in order to source ROS 1 for building the ros1_bridge (which is a ROS 2 package). Therefore I think we should probably switch the logic to "prefer" ROS 2 over ROS 1:

```suggestion
			ROS_VERSION: targetRos2Distro ? "2" : "1",
			ROS_PYTHON_VERSION:
				(targetRos2Distro || (targetRos1Distro && targetRos1Distro == "noetic")) ? "3" : "2",
```

This way, if we are using Galactic & Melodic, we get `ROS_VERSION=2` and `ROS_PYTHON_VERSION=3` instead of `ROS_VERSION=1` and `ROS_PYTHON_VERSION=2`. Does that make sense @kenji-miyake? There's probably no need to re-run all your tests after this change!

Reply:
Yes, it's fine! I'll commit your suggestion.

Reply:
Thank you! You'll just have to either squash all commits or add a sign-off to 8912a21d4aa732229acda8bade1f3b2e83d24ae7 so that the DCO job passes.

Reply:
Okay!

Reply:
Done?

Reply:
yep, it passed! Thank you!